### ISSUE TITLE
Whether --max_line_width and --max_line_count are not supported？

### ISSUE THREAD

**Issue by sulutian** [linto-ai/whisper-timestamped#184]

**Comment by Jeronymous** [comment]
Please provide a description.
Is it a request for a new feature / option for [make_subtitles.py](https://github.com/linto-ai/whisper-timestamped/blob/master/whisper_timestamped/make_subtitles.py#L112) ?

**Comment by sulutian** [comment]
> Please provide a description. Is it a request for a new feature / option for [make_subtitles.py](https://github.com/linto-ai/whisper-timestamped/blob/master/whisper_timestamped/make_subtitles.py#L112) ?

Yes, when there are parameters here, overly long sentences will not occur.
